Makefile

```diff
@@ -117,6 +117,7 @@ ifdef CI
 	make smb smb-windows
 	docker buildx build --no-cache --build-arg LDFLAGS=${LDFLAGS} -t $(IMAGE_TAG)-linux-amd64 -f ./pkg/smbplugin/Dockerfile --platform="linux/amd64" --push .
 	docker buildx build --no-cache --build-arg LDFLAGS=${LDFLAGS} -t $(IMAGE_TAG)-windows-1809-amd64 -f ./pkg/smbplugin/Windows.Dockerfile --platform="windows/amd64" --push .
+	docker buildx build --no-cache --build-arg LDFLAGS=${LDFLAGS} -t $(IMAGE_TAG)-linux-arm64 -f ./pkg/smbplugin/Dockerfile --platform="linux/arm64" --push .

```

Comment:
@andyzhangx I think this would do. But is `make smb-container` being called in the CI? I., the post image job?

Reply:
let's wait for the test result, I have credentials to push image to Azure Container Registry.

Reply:
I think it's better move L120 to L119

Reply:
Our Kubernetes ci pipeline supprts multiarch now including arm. But you need to setup the cloudbuild.yaml files like in the other csi repos